### PR TITLE
[AS9716-32d] Modify check eeprom via pre_pddf sh

### PR DESCRIPTION
**Pull request by jostar-yang**
Signed-off-by: Jostar Yang <jostar_yang@accton.com.tw>

#### Why I did it
Modify to use pre_pddf_init.sh to check eeprom is 0x57 or 0x56.
#### How I did it
Add pre_pddf_init.sh to code base.
#### How to verify it
Check DUT, access eeprom is fine
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

**Review by FuzailBrcm — DISMISSED** [review]

File: platform/broadcom/sonic-platform-modules-accton/as9716-32d/modules/pddf_custom_psu.c [hunk 1]
```diff
@@ -9,10 +9,99 @@
 #include <linux/sysfs.h>
 #include <linux/slab.h>
 #include <linux/dmi.h>
-#include "../../../../pddf/i2c/modules/include/pddf_psu_defs.h"
+#include "pddf_psu_defs.h"
 
 ssize_t pddf_show_custom_psu_v_out(struct device *dev, struct device_attribute *da, char *buf);
 extern PSU_SYSFS_ATTR_DATA access_psu_v_out;
+ssize_t pddf_get_custom_psu_model_name(struct device *dev, struct device_attribute *da, char *buf);
```
Comment (FuzailBrcm) [inline platform/broadcom/sonic-platform-modules-accton/as9716-32d/modules/pddf_custom_psu.c:16]: Are various types of PSUs applicable here for AS9716 platform?

Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as9716-32d/modules/pddf_custom_psu.c:16]: Current as9716 support ACBEL and YESM1300AM PSU. So other psu code can be moved. I will update code.

Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as9716-32d/modules/pddf_custom_psu.c:16]: Done to  remove not needed psu code.

**Comment by FuzailBrcm** [comment]
@jostar-yang 
Can you resolve this branch conflict and update the PR?

**Comment by jostar-yang** [comment]
Fix file conflict. Please check it.

**Review by FuzailBrcm — APPROVED** [review]

**Pull request merged**